### DISCUSSION TITLE
Is there a support for scheduled function execution yet?

### DISCUSSION THREAD

**Discussion by jlkiri** [vercel/vercel#4742] [Help] [answered]
Not sure if this was asked before.

**Comment by olaven** [comment, marked as answer]
I don't believe there is, but I would love to know if this is planned!

**Reply by styfle** [reply]
It is not supported and there is no ETA for this feature.

You can see some possible alternatives in #146.

In particular, this comment from Amio suggests GitHub Actions: https://github.com/vercel/vercel/issues/146#issuecomment-578723045

**Reply by olaven** [reply]
@styfle Thanks for your answer and handy links 😄
